kubernetes-discovery-client/src/main/java/io/micronaut/kubernetes/client/v1/KubernetesConfiguration.java

```diff
@@ -29,10 +29,7 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Map;
+import java.util.*;
```

Comment:
I would say it's not necessary to import whole `util` package due to `List`, is that IDE optimisation I guess, so could you please add just `import List ?`

Reply:
Yes, Intellij optimize imports^^. fixed.